include/of_controller.h

```diff
@@ -56,8 +78,17 @@ class OFController : public OFServer {
             switch_dpid_map(switch_dpid_map),
             port_id_map(port_id_map),
             OFServer(address, port, nthreads, secure,
-                     OFServerSettings().supported_version(4) // OF version 0x04 is OF 1.3
-                         .echo_interval(30)) { }
+                     OFServerSettings()
+                         .supported_version(4) // OF version 1 is OF 1.3
+                         .echo_interval(30)
+                         .keep_data_ownership(false)) {
+                             packet_in_counter = 0;
+                            //  ACA_LOG_INFO("%s\n", "Inited packet_in_counter to zero");
+                            std::cout<<"Inited packet_in_counter to zero"<<std::endl; 
+                               auto packet_in_counter_thread = new std::thread(std::bind(
```

Comment:
if this counter is only for testing, we can remove from the official PR

Reply:
Removed